### PR TITLE
Added spoilers to Message#getContentStripped

### PR DESCRIPTION
**Pull request by BenjaminUrquhart**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [X] I have checked the PRs for upcoming features/bug fixes.
- [X] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [X] Library interface (affecting end-user code) 
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: #906 

## Description

Message#getContentStripped should now strip spoiler tags

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/api/entities/Message.java [hunk 1]
```diff
@@ -344,9 +344,9 @@
     String getContentRaw();
 
     /**
-     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all markdown characters
-     * like {@literal *, **, __, ~~} that provide text formatting. Any characters that match these but are not being used
-     * for formatting are escaped to prevent possible formatting.
+     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all spoiler tags 
+     * and markdown characters like {@literal *, **, __, ~~, ||} that provide text formatting. Any characters that match these
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/entities/Message.java]: The spoiler tags are also markdown characters so this is redundant

File: src/main/java/net/dv8tion/jda/api/entities/Message.java [hunk 2]
```diff
@@ -344,9 +344,9 @@
     String getContentRaw();
 
     /**
-     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all markdown characters
-     * like {@literal *, **, __, ~~} that provide text formatting. Any characters that match these but are not being used
-     * for formatting are escaped to prevent possible formatting.
+     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all spoiler tags 
+     * and markdown characters like {@literal *, **, __, ~~, ||} that provide text formatting. Any characters that match these 
+     * but are not being used for formatting are escaped to prevent possible formatting.
```
Comment (MinnDevelopment) [inline src/main/java/net/dv8tion/jda/api/entities/Message.java]: Why was this changed?

File: src/main/java/net/dv8tion/jda/api/entities/Message.java (hunk 1 above)
Comment (BenjaminUrquhart) [reply src/main/java/net/dv8tion/jda/api/entities/Message.java]: Will revert

File: src/main/java/net/dv8tion/jda/api/entities/Message.java (hunk 2 above)
Comment (BenjaminUrquhart) [reply src/main/java/net/dv8tion/jda/api/entities/Message.java]: I tried to preserve the line length
